### PR TITLE
feat(core-ui, layouts, layouts-ui, ui): Update Flexbox Fly-Out Panel UI

### PR DESCRIPTION
**Pull request by VancheeZze**
## Changes

- Updated Component Picker width and height.
- Component Picker opens in the middle of the page and goes off the screen issue is fixed.
- Fixes issue with components preview were cut in half on the first ComponentPicker open.

## Test Instructions

- Run `npm run setup` first. It will update auto-generated `.css` files.
- Run `npm run start` and go to `/flexbox` page.
- Open Component picker and make sure `.rc-tooltip-content` div has right demensions. ( 714px x 744px ) 

- Add more content to the `Bodiless-JS/examples/test-site/src/data/pages/flexbox/index.tsx` so that you can scroll the page.
- Run `npm run start` and go to `/flexbox` page.
- Activate + icon by clicking on any flexbox areas and scroll page down.
- Try to open Component Picker -- it should stick to the top.

## Related Issues
Closes: https://github.com/johnsonandjohnson/Bodiless-JS/issues/54

### CODE REVIEW

File: packages/bodiless-layouts/src/ComponentSelector/index.tsx [hunk 1]
```diff
@@ -59,9 +59,6 @@ const ComponentSelector: React.FC<ComponentSelectorProps> = props => {
   return (
     <uiContext.Provider value={finalUI}>
       <finalUI.MasterWrapper>
```
Comment (VancheeZze) [inline packages/bodiless-layouts/src/ComponentSelector/index.tsx]: `SubmitButton ` was a duplicate. There is another one in the upper scope.

**Comment by dewen** [comment]
This PR is not created from fork?

**Comment by VancheeZze** [comment]
@dewen I just followed instructions in readme and cloned the repo. Worked off of it.

**Comment by dewen** [comment]
@VancheeZze I'm not sure if we need the option changes in package-lock.json. Please confirm you have the right process in updating this lock file. Thanks

**Pull request merged**